CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased] (beta)
 
 ### Added
+- Released the --sensitive-info flag, which will automatically obscure private data in logs.
```

Comment:
Can we make this `--mask-sensitive-data` instead?

Reply:
sure can.